src/plugins/intel_npu/src/plugin/npuw/base_sync_infer_request.cpp

```diff
@@ -583,6 +597,8 @@ void ov::npuw::IBaseInferRequest::bind_global_params(std::size_t idx, RqPtr requ
         bind_attention_inputs(idx, request);
     });
 
+    bind_pyramid_attention_inputs(idx, request);
```

Comment:
If this operation is costly and it make sense to track the time spent here, I'd recommend to add it under  the profiile (lines 596..598).